modules/honzon/src/weights.rs

```diff
@@ -55,97 +55,201 @@ pub trait WeightInfo {
 	fn close_loan_has_debit_by_dex() -> Weight;
 	fn expand_position_collateral() -> Weight;
 	fn shrink_position_debit() -> Weight;
+	fn precompile_get_current_collateral_ratio() -> Weight;
 }
 
 /// Weights for module_honzon using the Acala node and recommended hardware.
 pub struct AcalaWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> WeightInfo for AcalaWeight<T> {
+	// Storage: unknown [0x3a7472616e73616374696f6e5f6c6576656c3a] (r:1 w:1)
```

Comment:
any idea why is this storage unknown?

Reply:
I think it is from `T::Source::lookup()`, but can't quite figure out why it is unknown